content/shared/css/core.css

```diff
@@ -129,7 +129,52 @@ th .example-header {
   box-shadow: inset 0 3px 5px 1px hsl(216deg 82% 30%);
 }
 
+#support-notice[open] summary {
+  padding-bottom: 1em;
+  margin-bottom: 1em;
+}
+
 #support-notice summary p {
   display: inline;
   margin-left: 5px;
 }
+
+.note,
+.warning,
+.advisement {
+  border-right: none;
+  padding: 1em;
+  margin: 2em 0;
+}
+
+.note > h2,
+.note > h3,
+.note > h4,
+.warning > h2,
+.warning > h3,
+.warning > h4,
+.advisement > h2,
+.advisement > h3,
+.advisement > h4 {
```

Comment:
Should we consider specifying h5 even though we don't have any ... yet? There are a couple of practices, like keyboard, where we get down to h4 for subtopics.